cirq-core/cirq/ops/moment_test.py

```diff
@@ -617,3 +617,16 @@ def test_transform_qubits():
     assert original.transform_qubits(lambda q: cirq.GridQubit(10 + q.x, 20)) == modified
     with pytest.raises(TypeError, match='must be a function or dict'):
         _ = original.transform_qubits('bad arg')
+
+def test_single_gate_no_gates():
+    assert cirq.Moment() == None
+
+def test_single_gate_same_gates():
+    a = cirq.H(cirq.NamedQubit('a'))
+    b = cirq.H(cirq.NamedQubit('b'))
+    assert cirq.Moment([a, b]) == a
```

Comment:
Do you mean 

 `assert cirq.Moment([a, b]).get_single_gate_from_moment()  == a `

Reply:
Fixed.